src/js/otus/components/Detail/OTUHeaderEndIcons.tsx

```diff
@@ -11,6 +12,14 @@ type OTUHeaderEndIconsProps = {
     abbreviation: string;
 };
 
+const EditOTUIcon = styled(IconLink)`
+    margin: 0 5px;
+`;
+
+const RemoveOTUIcon = styled(IconLink)`
```

Comment:
Nitpick: I would just have a single `OTUIcon` since the style changes your applying are identical